sangchun-html/mypage.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/styles.css" />
+    <link rel="stylesheet" href="/css/sangchun/mypage.css" />
+    <title>Mypage</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="mypage-container">
+        <div class="nav-top">
+          <i class="fa-solid fa-chevron-left"></i>
+          <span>마이페이지</span>
+        </div>
+        <div class="mypage-mid">
+          <h4 id="mypage">개인정보 수정</h4>
+          <div class="mypage-click">
+            <div class="nick-container">
+              <span id="mypage-nick">닉네임 변경</span>
+              <span><i class="fa-solid fa-chevron-right"></i></span>
```

Comment:
```suggestion
              <a href="change-nickname.html"
                ><span><i class="fa-solid fa-chevron-right"></i></span
              ></a>
```
여기도 아이콘에 해당하는 페이지 링크 걸어주면 좋을 것 같아용